### ISSUE TITLE
[bug]: Hugo crashes when Details set to No

### ISSUE THREAD

**Issue by BenJamesBen** [freesewing/freesewing#6006]
### What seems to be the problem? 🤔

Steps to Reproduce:
1. On freesewing.org create a new Hugo using one of the curated sets.
2. In Core Settings -> Details, set to to Generate a Pattern Outline.
3. The crash occurs.

>ERROR TypeError: undefined is not an object (evaluating ‘g.facing.clone’)
ERROR Unable to draft part hugo.pocketFacing (set 0)
ERROR TypeError: undefined is not an object (evaluating ‘k.pocket.line’)
ERROR Unable to draft part hugo.pocket (set 0)

### Design / Plugin / Package 🧐

designs/hugo

**Comment by BenJamesBen** [comment]
```diff
--- a/designs/hugo/src/front.mjs
+++ b/designs/hugo/src/front.mjs
@@ -91,13 +91,13 @@ function hugoFront({
     .join(neckOpeningParts[0].reverse())
   paths.saBase.hide()
   paths.seam = paths.saBase.clone().unhide().close().attr('class', 'fabric')
-  if (complete)
-    paths.pocket = new Path()
-      .move(points.pocketHem)
-      .line(points.pocketTip)
-      .curve(points.pocketTip, points.pocketTopCp, points.pocketTop)
-      .line(points.pocketCf)
-      .attr('class', 'note help')
+  paths.pocket = new Path()
+    .move(points.pocketHem)
+    .line(points.pocketTip)
+    .curve(points.pocketTip, points.pocketTopCp, points.pocketTop)
+    .line(points.pocketCf)
+    .attr('class', 'note help')
+  if (!complete) paths.pocket.hide()
   // Store shoulder seam length, neck opening path, shoulder slope and raglan length
   store.set('shoulderLength', points.neck.dist(points.shoulder))
   store.set('neckOpeningPartFront', neckOpeningParts[1])
```

**Comment by joostdecock** [comment]
@BenJamesBen Probably this goes without saying, but you are free to put up a PR for these sort of things.

I've resolved it in #6009 but I went with:

```
.setHidden(!complete)
```

Since that avoids the `if` statement and we can just add it to the function chain.
